djvm/src/main/kotlin/sandbox/java/lang/DJVM.kt

```diff
@@ -241,6 +242,13 @@ fun getClassLoader(type: Class<*>): ClassLoader {
     return systemClassLoader
 }
 
+/**
+ * Replacement function for [ClassLoader.getSystemResourceAsStream].
+ */
+fun getSystemResourceAsStream(name: kotlin.String): InputStream? {
```

Comment:
Where is this function used other than in the "dummy" Currency.java?

Reply:
It's used in the _non-dummy_ `java.util.Currency$1`.